### ISSUE TITLE
v2.0.0 is no more compatible with Serverless v1

### ISSUE THREAD

**Issue by j0k3r** [claygregory/serverless-prune-plugin#37]
It happens when `this.log` is called from the plugin.
As `log` isn't injected in the plugin constructor (maybe it is in v2 or v3), it uses `serverless.cli.log`.
But `this.log` then calls `serverless.cli.log` (from CLI.js):

```js
  log(message, entity, opts) {
    const underline = opts ? opts.underline : false;
    const bold = opts ? opts.bold : false;
    const color = opts ? opts.color : null;

    let print = chalk.yellow;

    if (color) print = chalk.keyword(color);
    if (underline) print = print.underline;
    if (bold) print = print.bold;

    this.consoleLog(`${entity || 'Serverless'}: ${print(message)}`);
  }
```

In that context, it seems `this` is affected to `Prune` so `this.consoleLog` does not exist.
To fix it, I guess the faster will be to create a `consoleLog` method in the Prune class, should I submit a PR then?

Here is the error.

```
  TypeError: this.consoleLog is not a function
      at Prune.log (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/lib/classes/CLI.js:414:10)
      at Prune.logNotice (/Users/j0k/Sites/20mn/heimdall/node_modules/serverless-prune-plugin/index.js:348:15)
      at Prune.cliPrune (/Users/j0k/Sites/20mn/heimdall/node_modules/serverless-prune-plugin/index.js:102:12)
      at /Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/lib/classes/PluginManager.js:498:55
      at tryCatcher (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/util.js:16:23)
      at Object.gotValue (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/reduce.js:168:18)
      at Object.gotAccum (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/reduce.js:155:25)
      at Object.tryCatcher (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:547:31)
      at Promise._settlePromise (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromiseCtx (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:641:10)
      at _drainQueueStep (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:97:12)
      at _drainQueue (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:86:9)
      at Async._drainQueues (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:102:5)
      at Immediate.Async.drainQueues [as _onImmediate] (/Users/j0k/.nvm/versions/node/v12.22.7/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:15:14)
      at processImmediate (internal/timers.js:461:21)
```

**Comment by claygregory** [comment]
Hi @j0k3r — oh my, this was a horrible oversight on my part. 

```javascript
this.log = log || serverless.cli.log;
```
should be
```javascript
this.log = log || serverless.cli.log.bind(serverless.cli);
```

Thanks so much for the report. This should be resolved in 2.0.1 — please give it a try.

**Comment by j0k3r** [comment]
Thanks @claygregory, fixed!
